Entity/User.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/**
+ *
+ * @package phpBBSessionsAuthBundle
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license MIT
+ *
+ */
+namespace phpBB\SessionsAuthBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Security\Core\Role\Role;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * Class User
+ * @package phpbb\SessionsAuthBundle\Entity
+ * @ORM\Entity(readOnly=true)
```

Comment:
Well, I am kinda a genuis :) It won't however prevent from creating new ones, but that should be fine.